recipes/watch.mdwn

```diff
@@ -146,6 +146,24 @@ local mpris, mpris_timer = awful.widget.watch(
 )
 ```
 
+## pipewire
+```lua
```

Comment:
I think we need an extra empty line here to allow the markdown interpreter to work :

```suggestion
## pipewire

```lua
```